internal/service/cognitoidp/user_pool_test.go

```diff
@@ -603,6 +603,35 @@ func TestAccCognitoIDPUserPool_sms(t *testing.T) {
 	})
 }
 
+func TestAccCognitoIDPUserPool_SMS_snsRegion(t *testing.T) {
+	rName := sdkacctest.RandomWithPrefix(acctest.ResourcePrefix)
+	iamRoleResourceName := "aws_iam_role.test"
+	resourceName := "aws_cognito_user_pool.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t); testAccPreCheckIdentityProvider(t) },
+		ErrorCheck:               acctest.ErrorCheck(t, cognitoidentityprovider.EndpointsID),
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories,
+		CheckDestroy:             testAccCheckUserPoolDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccUserPoolConfig_smsConfigurationSnsRegion(rName, "us-east-1"),
```

Comment:
```suggestion
				Config: testAccUserPoolConfig_smsConfigurationSnsRegion(rName, acctest.Region()),
```
This avoids hard-coding the region name in the tests. As per the [docs](https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/running-and-writing-acceptance-tests.md#running-an-acceptance-test), the tests default to `us-west-2` but can be overridden via env vars/aws profile configs. When I initially ran these tests, the API complained that `InvalidParameterException: Invalid snsRegion. Allowed SNS region for us-west-2 is us-west-2`.